some fixing and optimize image assets.

saurabharch opened the pull request.

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 2,576.45kb -> 716.67kb (72.18%)

/packages/editor-ui/src/plugins/i18n/docs/img/cred.png -- 857.43kb -> 116.03kb (86.47%)
/packages/editor-ui/src/plugins/i18n/docs/img/header5.png -- 296.37kb -> 56.59kb (80.91%)
/packages/editor-ui/src/plugins/i18n/docs/img/header4.png -- 344.11kb -> 72.81kb (78.84%)
/packages/editor-ui/src/plugins/i18n/docs/img/header2.png -- 160.65kb -> 34.53kb (78.5%)
/packages/editor-ui/src/plugins/i18n/docs/img/header1.png -- 252.76kb -> 57.42kb (77.28%)
/packages/editor-ui/src/plugins/i18n/docs/img/header3.png -- 238.65kb -> 68.84kb (71.15%)
/packages/editor-ui/src/plugins/i18n/docs/img/node4.png -- 82.33kb -> 25.84kb (68.62%)
/packages/editor-ui/src/plugins/i18n/docs/img/node1.png -- 38.05kb -> 22.81kb (40.04%)
/packages/editor-ui/public/google-signin-light.png -- 7.87kb -> 4.75kb (39.68%)
/packages/nodes-base/nodes/Marketstack/marketstack.svg -- 3.17kb -> 1.93kb (39.15%)
/packages/nodes-base/nodes/FreshworksCrm/freshworksCrm.svg -- 4.85kb -> 3.54kb (26.92%)
/packages/nodes-base/nodes/Grafana/grafana.svg -- 10.11kb -> 7.55kb (25.33%)
/packages/editor-ui/src/plugins/i18n/docs/img/node2.png -- 50.03kb -> 38.42kb (23.21%)
/packages/nodes-base/nodes/Coda/coda.svg -- 0.69kb -> 0.54kb (21.45%)
/packages/nodes-base/nodes/Mautic/mautic.svg -- 1.36kb -> 1.08kb (20.27%)
/assets/n8n-screenshot.png -- 126.66kb -> 105.55kb (16.66%)
/packages/nodes-base/nodes/Elastic/Elasticsearch/elasticsearch.svg -- 1.37kb -> 1.15kb (15.73%)
/packages/nodes-base/nodes/Google/YouTube/youTube.png -- 1.01kb -> 0.86kb (15.51%)
/packages/nodes-base/nodes/MonicaCrm/monicaCrm.png -- 3.91kb -> 3.38kb (13.68%)
/packages/nodes-base/nodes/Elastic/ElasticSecurity/elasticSecurity.svg -- 0.91kb -> 0.78kb (13.67%)
/packages/nodes-base/nodes/Magento/magento.svg -- 0.66kb -> 0.57kb (12.78%)
/packages/nodes-base/nodes/Microsoft/GraphSecurity/microsoftGraph.svg -- 2.23kb -> 1.96kb (12.09%)
/packages/nodes-base/nodes/WooCommerce/wooCommerce.svg -- 3.34kb -> 3.02kb (9.6%)
/packages/nodes-base/nodes/QuestDb/questdb.png -- 6.13kb -> 5.61kb (8.46%)
/packages/nodes-base/nodes/Brandfetch/brandfetch.png -- 0.54kb -> 0.51kb (5.77%)
/packages/nodes-base/nodes/UProc/uproc.png -- 1.94kb -> 1.85kb (5.07%)
/packages/nodes-base/nodes/Stripe/stripe.svg -- 1.61kb -> 1.53kb (4.79%)
/packages/editor-ui/public/n8n-icon-small.png -- 1.13kb -> 1.09kb (4.13%)
/packages/nodes-base/nodes/Aws/DynamoDB/dynamodb.svg -- 2.16kb -> 2.08kb (3.84%)
/packages/nodes-base/nodes/CrateDb/cratedb.png -- 0.46kb -> 0.45kb (2.32%)
/packages/nodes-base/nodes/HomeAssistant/homeAssistant.svg -- 5.17kb -> 5.09kb (1.51%)
/packages/nodes-base/nodes/Pipedrive/pipedrive.svg -- 2.10kb -> 2.07kb (1.3%)
/packages/nodes-base/nodes/N8nTrainingCustomerDatastore/n8nTrainingCustomerDatastore.svg -- 1.72kb -> 1.70kb (0.8%)
/packages/nodes-base/nodes/Zendesk/zendesk.svg -- 0.63kb -> 0.63kb (0.77%)
/packages/nodes-base/nodes/Microsoft/Dynamics/dynamicsCrm.svg -- 1.14kb -> 1.13kb (0.69%)
/packages/nodes-base/nodes/RespondToWebhook/webhook.svg -- 0.94kb -> 0.93kb (0.63%)
/packages/nodes-base/nodes/NocoDB/nocodb.svg -- 32.09kb -> 31.92kb (0.52%)
/packages/nodes-base/nodes/Dropcontact/dropcontact.svg -- 0.75kb -> 0.75kb (0.39%)
/packages/nodes-base/nodes/N8nTrainingCustomerMessenger/n8nTrainingCustomerMessenger.svg -- 1.58kb -> 1.58kb (0.25%)
/packages/nodes-base/nodes/Grist/grist.svg -- 20.64kb -> 20.59kb (0.21%)
/packages/nodes-base/nodes/ProfitWell/profitwell.png -- 1.83kb -> 1.82kb (0.16%)
/packages/nodes-base/nodes/ActionNetwork/actionNetwork.svg -- 1.77kb -> 1.77kb (0.11%)
/packages/nodes-base/credentials/AWS.svg -- 3.63kb -> 3.63kb (0.03%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

saurabharch committed:
Merge pull request #1 from saurabharch/imgbot

[ImgBot] Optimize images

saurabharch committed:
Merge branch 'n8n-io:master' into master

saurabharch committed:
:wheelchair: update .gitignore declaration

saurabharch committed:
:bug: Fixing Zulip Network Api Request.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/n8n-io/n8n?pullRequest=2925) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/n8n-io/n8n?pullRequest=2925) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: saurabharch<br/>:x: ImgBotApp<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/n8n-io/n8n?pullRequest=2925) it.</sub>

saurabharch committed:
:bug: zoom netwrok error during api required is now fixed

saurabharch committed:
:bug: Fixing ZohoCrm Node Network Api request.

saurabharch committed:
:bug: Fixing Zendesk Network Api Request Error.

saurabharch committed:
Fixed Lint Issue

saurabharch committed:
:bug: Fixing Xero Network api request error and fix zammad lint

saurabharch committed:
:bug: Update Code with Release onces.

saurabharch committed:
:bug:  Fix Wordpress Network Api Request Error.

saurabharch committed:
:bug: Fixing Network Call Api Request Error.

saurabharch committed:
:bug: Fixing Network Api Call Request Error.

saurabharch committed:
:bug: Fixing Woocomerece Api call request Error.

saurabharch committed:
:bug: Fixing Vero Network api call request Error.

saurabharch committed:
:bug: Fixing UrlScamIo Network call api Request Error.

saurabharch committed:
:bug: Fixing UptimeRobot Network call Api Request.

saurabharch committed:
:bug: Fixing UProc Network call Api Request Error.

saurabharch committed:
:bug: Fixing Uplead Network call Api Request Error.

saurabharch committed:
:bug: Fixing Unleashed Software Network Api call request Error.

saurabharch committed:
:bug: Fixing Twitter Network Call Api Request Error.

saurabharch committed:
:bug: Fixing Twitst Network Api call request Error.

saurabharch committed:
:bug: Fixing Twillio Network api call request Error.

saurabharch committed:
:bug: Fixing Tweka Network HTTP request call Error.

saurabharch committed:
:bug: Fixing Trello Network Api call Request Error.

saurabharch committed:
:bug: Fixing TravisCi Network call api Request Error.

saurabharch committed:
:bug: Fixing ToggleTrigger Api call request Error.

saurabharch committed:
:bug: Fixing Todoist Api Network call request Error.

saurabharch committed:
:bug: Fixing Telegram Network call Api Request Error.

saurabharch committed:
:bug: Fixing Tapfilliate Network call Api Request Error.

saurabharch committed:
:bug: Fixing Taiga Network Api call request Error.

saurabharch committed:
:bug: Fixing WufooTriger Network Api call request Error.

saurabharch committed:
:bug: Fixing SyncroMsp Network api call request Error.

saurabharch committed:
:bug: Fixing SurveyMonkey Api Network Call request Error.

saurabharch committed:
:bug: Update SurveyMonkey Network Error Request with Node Error callback

saurabharch committed:
:bug: Fixing Stripe Network Api call request Error.

saurabharch committed:
:bug: Fixing Strava Network Api call request Error.

saurabharch committed:
:bug: Fixing Storyblok Network api call request Error.

saurabharch committed:
:bug: Fixing Stackby Network api call request Error.

saurabharch committed:
:bug:  Fixing Spotify Network Api call request Error.

saurabharch committed:
:bug: Fixing Spontit Network api call request Error.

saurabharch committed:
:bug: Fixing Lint errors

saurabharch commented:
node base optimization for public network api call request handle.

snyk-bot committed:
fix: upgrade typeorm from 0.2.30 to 0.2.43

Snyk has created this PR to upgrade typeorm from 0.2.30 to 0.2.43.

See this package in npm:
https://www.npmjs.com/package/typeorm

See this project in Snyk:
https://app.snyk.io/org/saurabharch/project/dc5209e7-436b-4e54-9443-cc95b4e801d5?utm_source=github&utm_medium=referral&page=upgrade-pr

snyk-bot committed:
fix: upgrade mqtt from 4.2.6 to 4.3.6

Snyk has created this PR to upgrade mqtt from 4.2.6 to 4.3.6.

See this package in npm:
https://www.npmjs.com/package/mqtt

See this project in Snyk:
https://app.snyk.io/org/saurabharch/project/a70fb506-30f9-4503-9007-578daf195a51?utm_source=github&utm_medium=referral&page=upgrade-pr

snyk-bot committed:
fix: upgrade json-diff from 0.5.5 to 0.7.1

Snyk has created this PR to upgrade json-diff from 0.5.5 to 0.7.1.

See this package in npm:
https://www.npmjs.com/package/json-diff

See this project in Snyk:
https://app.snyk.io/org/saurabharch/project/dc5209e7-436b-4e54-9443-cc95b4e801d5?utm_source=github&utm_medium=referral&page=upgrade-pr

snyk-bot committed:
fix: upgrade @types/json-diff from 0.5.2 to 0.7.0

Snyk has created this PR to upgrade @types/json-diff from 0.5.2 to 0.7.0.

See this package in npm:
https://www.npmjs.com/package/@types/json-diff

See this project in Snyk:
https://app.snyk.io/org/saurabharch/project/dc5209e7-436b-4e54-9443-cc95b4e801d5?utm_source=github&utm_medium=referral&page=upgrade-pr

saurabharch committed:
Merge pull request #21 from saurabharch/snyk-upgrade-bda852ac563b38ce6de694754e555cdd

[Snyk] Upgrade @types/json-diff from 0.5.2 to 0.7.0

saurabharch committed:
Merge pull request #18 from saurabharch/snyk-upgrade-79bc5160c0d2c9184d185bb5dd21bac1

[Snyk] Upgrade typeorm from 0.2.30 to 0.2.43

snyk-bot committed:
fix: packages/nodes-base/package.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-SIMPLEGIT-2421199

saurabharch committed:
Merge pull request #20 from saurabharch/snyk-upgrade-5b30a06a5622ac3be9b918f46671e902

[Snyk] Upgrade json-diff from 0.5.5 to 0.7.1

saurabharch committed:
Merge pull request #22 from saurabharch/snyk-fix-d07dcd2bce45954adb9e019c9cbe1a56

[Snyk] Security upgrade simple-git from 2.48.0 to 3.3.0

saurabharch committed:
Merge pull request #19 from saurabharch/snyk-upgrade-53f120ff0e4d41be75c638ba6e670284

[Snyk] Upgrade mqtt from 4.2.6 to 4.3.6
